package-lock.json

```diff
@@ -1503,7 +1503,7 @@
         },
         "chalk": {
           "version": "1.1.3",
-          "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
+          "resolved": "http://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
```

Comment:
@kellgauthier This is generally not a thing you want to see (https to http). Also, since this update to package-lock.json is not relevant to the work, please don't remove it from this PR. Thanks!